### PR TITLE
sticky nav cleanup

### PR DESCRIPTION
**Pull request by shycoder-06**
cleanup

1. nesting removed and replaced with separate links
2. hover added using css
3. margin for nav and bottom contents placed in one class for scalability
4. opacity for background image
5. updated bg color

outstanding issue

1. the text overlay on image is on the back, i m unable to bring it to the front (tried z-index but still unsuccesful)
3. although i used Open Sans, some fonts look a bit off

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,69 +1,87 @@
...
+  padding-bottom: 50px;
 }
 
-.nav-bar .nav-container {
-  background-color: #000000;
-  color: #ffffff;
+.page-bg {
+  background-image: url("https://images.pexels.com/photos/450035/pexels-photo-450035.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+  height: 100vh;
+  width: 100vw;
```
Comment (sethun60) [inline style.css:14]: This is not needed. This makes the screen look odd. You can see that there is gap in opacity and the actual image. 

Screen Shot 2022-11-22 at 8 26 17 AM

File: style.css [hunk 2]
```diff
@@ -1,69 +1,87 @@
...
-.nav-container {
-  padding: 10px 0px;
+.header {
+  z-index: -1;
+}
+/* Applied to nav and content to maintain padding and margins equally */
+.space {
+  padding: 20px 30px;
+  max-width: 1200px;
+  margin: 0 100px;
```
Comment (sethun60) [inline style.css]: this should be set to margin auto

File: style.css [hunk 3]
```diff
@@ -1,69 +1,87 @@
...
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
 }
-
-.nav-container {
-  padding: 10px 0px;
+.header {
+  z-index: -1;
```
Comment (sethun60) [inline style.css:29]: This is not needed. You shouldn't use z-index like this. Z-indices should be used carefully.

**Review by sethun60 — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (shycoder-06) [reply style.css:14]: fixed.

**Review by sethun60 — APPROVED** [review]
LGTM.

**Comment by sethun60** [comment]
Please merge and delete the branch.

**Comment by shycoder-06** [comment]
merged PR and deleted branch.

**Pull request merged**